cards/src/main/java/com/handonn/finapp/cards/controller/CardController.java

```diff
@@ -0,0 +1,72 @@
+package com.handonn.finapp.cards.controller;
+
+import com.handonn.finapp.cards.model.CardDto;
+import com.handonn.finapp.cards.service.ICardService;
+import com.handonn.finapp.common.model.BaseResponse;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.Pattern;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.HttpStatus;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping(path = "/api/v1/cards")
+@RequiredArgsConstructor
+@Validated
+public class CardController {
+
+    private final ICardService cardService;
+
+    @PostMapping
+    public BaseResponse<?> create(@Valid @RequestBody CardDto cardDto) {
+        cardService.create(cardDto);
+        return BaseResponse.builder()
+                .statusCode(HttpStatus.CREATED)
+                .message("Created")
+                .build();
+    }
+
+    @GetMapping
+    public BaseResponse<List<CardDto>> getCardList(Pageable pageable) {
```

Comment:
Params should be getCardList(int offset, int size)
Then convert it to Pageable in the Service layer